components/Layout.tsx

```diff
@@ -10,7 +9,7 @@ import NavBar from './NavBar'
 
 const defaultMetaTags = {
     description: 'Find and fix things across all of your code with Sourcegraph universal code search.',
-    image: 'https://about.sourcegraph.com/sourcegraph-mark.png',
+    image: '/headers/sourcegraph-learn-header.png',
```

Comment:
Do you think we should have a full path with `https://` here? On Firefox I am getting a little noise about mixed security levels and think the full path may be a solution

Reply:
A bit further down the url gets the full path added with `https`, so the relative path should be OK to use.

https://github.com/sourcegraph/learn/blob/96d56daa1e621868f310b94f385ef895f48867d8/components/Layout.tsx#L49-L51

I see a mixed security warning too, but I think it might have to do with the`http` image for the video thumbnail. I added a change to make it `https` and we can check if that fixes it.

Reply:
ok, cool, thanks